web_widget_map/README.rst

```diff
@@ -0,0 +1,99 @@
+==============
+Web Widget Map
+==============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_widget_map
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_map
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module add the possibility to insert a map into Odoo form views using
+the OpenLayers library.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To insert a Bokeh chart in a view proceed as follows:
+
+#. Add the widget in the view::
+
+    <widget name='map_widget'/>
+
+
+When the widget is set in a view, the user will see a button, clicking on
+the button will render the map.
+
+.. figure:: https://raw.githubusercontent.com/OCA/web/12.0/web_widget_map/static/src/img/view_map_button.png
+    :align: center
+    :width: 600 px
+
+.. figure:: https://raw.githubusercontent.com/OCA/web/12.0/web_widget_map/static/src/img/view_map.png
+    :align: center
+    :width: 600 px
+
+By default the center of the map is set to latitude 0 and longitude 0.
+Refer to the module hr_attendance_geolocation_map to see an example on how
```

Comment:
greats to have an exemple, without refering on a another module.
or at least, set an url to find easily this module. (https://github.com/OCA/hr-attendance)

regards.